packages/wpcom-checkout/src/can-item-be-removed-from-cart.ts

```diff
@@ -31,13 +31,13 @@ export function canItemBeRemovedFromCart(
 	item: ResponseCartProduct,
 	responseCart: ResponseCart
 ): boolean {
-	const itemTypesThatCannotBeDeleted = [ 'domain_redemption' ];
+	const itemTypesThatCannotBeDeleted = [ 'domain_redemption', 'wp_difm_extra_page' ];
 	if ( itemTypesThatCannotBeDeleted.includes( item.product_slug ) ) {
 		return false;
 	}
 
-	// The Premium plan cannot be removed from the cart when in combination with the DIFM lite product
-	if ( isPremiumPlanWithDIFMInTheCart( item, responseCart ) ) {
+	// The Pro plan cannot be removed from the cart when in combination with the DIFM lite product
+	if ( isProPlanWithDIFMInTheCart( item, responseCart ) ) {
```

Comment:
The PR title or description doesn't mention anything about changes to what products can be removed from the cart. Neither does the commit. Can you say more about this part of the change?

Reply:
This was just changing Premium -> Pro which needs to revert back now that we are back on premium plans 😅 .
Thanks for raising this!